# expected: RecognitionException
const a
a=5
